### PR TITLE
KubeVirt opinionated deployment via Hyperconverged Cluster Operator

### PR DESCRIPTION
**Pull request by tiraboschi**

### CODE REVIEW

File: events/2021-kubevirt-summit/proposals/hco-user-proposal-2021.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+* Node Maintenance
+
+The purpose of HCO is to ease the deployment, upgrade, monitoring and configuration of an opinionated version of the KubeVirt cluster.
+
+The Hyperconverged Cluster Operator can be installed on bare metal server clusters in a matter of minutes, even from a GUI, without requiring a deep knowledge of Kubernetes internals. 
+
+An attendee will learn:
+* how to deploy and maintain a KubeVirt cluster with the Hyperconverged Cluster Operator
+* demo
+* ongoing development and how to contribute
```
Comment (codificat) [inline events/2021-kubevirt-summit/proposals/hco-user-proposal-2021.md:21]: maybe the development aspects around HCO should be covered in #54 ?

**Review by codificat — COMMENTED** [review]
Thanks for submitting the proposal!

There is another proposal about HCO, #54, for the contributors track (i.e. aimed at contributors rather than users). That should be fine, as the contents and expected audience and goals would be different.

However, right now the two submissions as they are look maybe too similar? Could you work with #54 to clarify the scope of each, and/or explore the option to sync/

**Comment by codificat** [comment]
Thank you for submitting this session proposal for the KubeVirt summit!

We are working to prepare a schedule that can accommodate as many of the received proposals as possible, according to the capacity available in the event.

We might have to request some adjustments to some of the proposals in order to achieve that goal.

More feedback will be provided next week.

Looking forward to meeting you at the event!

**Review by davidvossel — COMMENTED** [review]
Due to having received more content than we expected, we had to quickly form a review team (thanks Red Hat and NVIDIA) to make difficult decisions about what presentations to accept.

**This submission is currently on a wait list,** which means that it’s possible that it will be accepted if an opening occurs as the schedule is being formed.

As a result, we’d like to keep this PR open in the event that someone backs out. Thank you for your submission and support of the KubeVirt community!

**Comment by codificat** [comment]
As we finalize the schedule we now have an available slot.

**This session has been accepted!**

We ask that presentation slides be submitted by Friday, Feb 5th 2021.

More information pertaining to the event schedule, event platform, and how to submit slides will be made available to you soon. If at any point you find that you are unable to present, please let us know as soon as possible so we can fill your slot from our wait list.

Thank you for your submission and support of the KubeVirt Community.

**Pull request merged**